src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLHTMLOutput.java

```diff
@@ -127,6 +122,7 @@ public String toHTML() {
                 .append("mapml-viewer:not(:defined) > * { display: none; } n")
                 .append("map-layer { display: none; }\n")
                 .append("</style>\n")
+                .append(buildProjectionScript(projType))
```

Comment:
Better to pass in the stringbuilder and allow appending stuff as needed inside, no?